Tests/FileTest.php

```diff
@@ -14,6 +14,60 @@
  */
 class FileTest extends FilesystemTestCase
 {
+    /**
+     * Provides the data to test the getExt method.
+     *
+     * @return  \Generator
+     */
+    public function dataTestGetExt(): \Generator
+    {
+        yield [
+            'foobar.php',
+            'php',
+        ];
+
+        yield [
+            'foobar..php',
+            'php',
+        ];
+
+        yield [
+            'foobar.php.',
+            '',
+        ];
+
+        yield [
+            'foobar.zip',
+            'zip',
+        ];
+        yield [
```

Comment:
```suggestion

        yield [
```
Sorry, my mistake. Wanted to suggest removal of one of the 2 empty lines but GitHub made it to remove both.